app/controllers/application_controller.rb

```diff
@@ -38,4 +38,8 @@ def force_www!
   def current_event
     @current_event = Event.current
   end
+
+  def nav_events
+    @nav_events = Event.upcoming_events.public_events.order('start_date').take(4)
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.